src/test/java/bio/terra/service/dataset/DatasetBucketDaoTest.java

```diff
@@ -25,25 +25,22 @@
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.junit4.SpringRunner;
 
-@RunWith(SpringRunner.class)
-@SpringBootTest
-@AutoConfigureMockMvc
 @ActiveProfiles({"google", "unittest"})
-@Category(Unit.class)
+@AutoConfigureMockMvc
 @EmbeddedDatabaseTest
+@SpringBootTest
+@Tag("bio.terra.common.category.Unit")
 public class DatasetBucketDaoTest {
```

Comment:
Test category: "unit" test with an embedded database.  (Ideally, our unit tests wouldn't talk to databases and database tests would be classified separately.)

I am puzzled by the need for `@AutoConfigureMockMvc` in this kind of test.  I can't get initialization to work without it.  I also couldn't get test slicing to work.